ui/app/pages/home/home.component.js

```diff
@@ -157,11 +161,49 @@ export default class Home extends PureComponent {
       </MultipleNotifications>
     )
   }
+  renderPopover = () => {
+    const { setConnectedStatusPopoverHasBeenShown } = this.props
+    const { t } = this.context
+    return (
+      <Popover
+        title={ t('whatsthis') }
+        onClose={() => setConnectedStatusPopoverHasBeenShown()}
```

Comment:
Nit: could you pass in this function directly, rather than wrapped in an arrow function? The same below for the "dismiss" button `onClose` prop

Reply:
Done in 593fdffa2